packages/testing/src/jest-shim.ts

```diff
@@ -57,6 +57,34 @@ window.focus = () => {
   /* JSDom throws "Not Implemented" */
 };
 
+/**
+ * Shim scrollTo because jsdom still does not implement a shim for it, see:
+ * https://github.com/jsdom/jsdom/issues/2751
+ */
+window.Element.prototype.scrollTo = (
+  _optionsOrX?: ScrollToOptions | number
+) => {
+  // no-op
+};
+
+// https://github.com/jsdom/jsdom/issues/3368
+class ResizeObserverMock {
+  constructor(_callback: any) {
+    // no-op
+  }
+  observe(_target: any, _options?: any) {
+    // no-op
+  }
+  unobserve(_target: any) {
+    // no-op
+  }
+  disconnect() {
+    // no-op
+  }
+}
+
+window.ResizeObserver = ResizeObserverMock;
```

Comment:
Also checking my understanding here. If the ResizeObserver is basically just a no-op mock, that means that none of the tests are hitting code paths that would lead to _scrollBackToItemOnResize() being called, correct?

Reply:
> that means that none of the tests are hitting code paths

More precisely none of the _unit_ tests get to call `_scrollBackToItemOnResize()`, but the integration tests do. In fact if one was to remove it the visual tests would fail.